packages/playground/src/api/documents/previewApi.ts

```diff
@@ -0,0 +1,68 @@
+// Copyright 2020 Cognite AS
+
+import { BaseResourceAPI } from '@cognite/sdk-core';
+import { HttpResponseType } from '@cognite/sdk-core';
+
+import { DocumentId, DocumentPreviewTemporaryLink } from '../../types';
+import Any = jasmine.Any;
+
+export class PreviewAPI extends BaseResourceAPI<Any> {
+  public documentAsPdf = (id: DocumentId): Promise<ArrayBuffer> => {
```

Comment:
@andersfylling I am not familiar with the coding conventions used in the sdk, but IMO we shoudn't use `public` since by default class functions are public anyway.
Also I would change functions to just plain functions instead of property = function.

```JS
documentAsPdf(id: DocumentId): Promise<ArrayBuffer>  {
//...code here
}
```

Reply:
@bdimitrijoski regarding arrow functions: we have it that way in other APIs, makes sense to keep consistency